activerecord/lib/active_record/connection_adapters/postgresql/oid/serial.rb

```diff
@@ -0,0 +1,10 @@
+module ActiveRecord
+  module ConnectionAdapters
+    module PostgreSQL
+      module OID # :nodoc:
+        class Serial < Integer # :nodoc:
+        end
+      end
+    end
+  end
+end
```

Comment:
There's no magic going on with the type system or attributes API, if you don't see something referenced in the code, it won't be used.